torch/_dynamo/trace_rules.py

```diff
@@ -0,0 +1,48 @@
+import enum
+import functools
+import importlib
+
+
+class TraceRule(enum.Enum):
+    IN_GRAPH_FUNCTION = 0
+    CONST_FOLD_FUNCTION = 1
+    SUPPORTED_CTX_MANAGER_CLASS = 2
+    SKIP = 3
+    INLINE = 4
+
+
+torch_name_rule_map = {
```

Comment:
Need a doc block explaining what this is, how we check it is complete, and how to update it.